src/main/java/edu/kis/vh/nursery/IntArrayStack.java

```diff
@@ -0,0 +1,42 @@
+package edu.kis.vh.nursery;
+// kombinacja alt + <- lub alt + -> powoduje przełączanie się między zakładakmi (otwartymi klasami)
+public class IntArrayStack {
+	private static final int FULL = 11;
```

Comment:
This constant is related to STACK_CAPACITY. Changing only one leads to an error (inconsistency).